docs/1-Applying/how_to_apply.md

```diff
@@ -40,4 +44,6 @@ If a friend can refer to you a company (maybe a company they've worked) this can
 
 However - I tend to not ask for referals or give them because I don't work technically with many of my friends. I can vouch for them being kind and thoughtful and hard working (and same for them about me) but neither of us really know how well the other works on technical things and so unless they're a close friend I'd probably feel weird referring them.
 
-What I *have* done (and asked for) is introduced my friends to recruiters, in sort of a "here's my friend they're pretty cool and also interested in your company" and less of a "I recommend this person to work here" kind of way. Maybe this is what most referals are anyways, but there you go.
+What I *have* done (and asked for) is introduced my friends to recruiters, in sort of a "here's my friend they're pretty cool and also interested in your company" and less of a "I recommend this person to work here" kind of way. Even if the most your friend can say about you is that you're a nice person, this is still valuable. Very broadly speaking, most companies look for two things: "this person is technically competent" and "this person will work effectively in a team" (i.e. a fancy way of saying "not an asshole"). A non-technical referral is still useful for the recruiter since they don't need to worry about the latter as much.
+
+Of course, for a friend to introduce you as cool person, this requires you to be a cool person. This is a reputation you build throughout your time as a student. Be nice to everyone you meet, not just the ones who can do you favors! You never know who might end up in a position to give referrals at a desirable company.
```

Comment:
> Be nice to everyone you meet

whoa whoa whoa - although being kind is often good, I think there are plenty of times when it's reasonable to be a jerk to someone.

idk this paragraph feels weird and a bit scummy to me. 

maybe:

> Of course, for someone to introduce you as someone good to work with, they need to think you're good to work with! This is a reputation you build throughout your time as a student. Be kind and helpful when you can, and not just to the ones who can do you favors! You never know who might end up in a position to give you an intro to a company you're interested in.